OpenRA.Mods.TS/Activities/VoxelHarvesterDockSequence.cs

```diff
@@ -17,24 +18,52 @@ namespace OpenRA.Mods.TS.Activities
 	public class VoxelHarvesterDockSequence : HarvesterDockSequence
 	{
 		readonly WithVoxelUnloadBody body;
+		readonly WithHarvesterUnloadOverlay spriteOverlay;
```

Comment:
That trait should be renamed to `WithDockedOverlay`. It is named wrong because responds to harvester *docked* events, not *docking* events.